src/view/index.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://cdn.jsdelivr.net/npm/react-pdf@9.1.0/dist/esm/Page/AnnotationLayer.css');
```

Comment:
Why not importing the one from node_modules?
import 'react-pdf/dist/esm/Page/AnnotationLayer.css';